compiler/rustc_metadata/src/rmeta/decoder/cstore_impl.rs

```diff
@@ -226,15 +226,7 @@ provide! { tcx, def_id, other, cdata,
     lookup_default_body_stability => { table }
     lookup_deprecation_entry => { table }
     params_in_repr => { table }
-    // FIXME: Could be defaulted, but `LazyValue<UnusedGenericParams>` is not `FixedSizeEncoding`..
-    unused_generic_params => {
-        cdata
-            .root
-            .tables
-            .unused_generic_params
-            .get(cdata, def_id.index)
-            .map_or_else(|| ty::UnusedGenericParams::new_all_used(), |lazy| lazy.decode((cdata, tcx)))
-    }
+    unused_generic_params => { cdata.root.tables.unused_generic_params.get(cdata, def_id.index) }
```

Comment:
```suggestion
    unused_generic_params => { table_direct }
```

I think?

Reply:
This would require `impl ProcessQueryValue for UnusedGenericParams`, which is 7 lines of trouble more than it's worth imo